docs/uptime/index.asciidoc

```diff
@@ -0,0 +1,23 @@
+[role="xpack"]
+[[xpack-uptime]]
+= Uptime
+
+Use the Uptime UI to monitor the status of network endpoints via HTTP/S, TCP,
+and ICMP. You will be able to explore status over time, drill into specific monitors,
+and view a high-level snapshot of your environment at a selected point in time.
+
+== Add monitors
+You define your monitors in Heartbeat's configuration file. Refer to the Heartbeat
+documentation for instructions on configuring monitors to begin storing Uptime information
+in your cluster.
+
+== Begin using Uptime
```

Comment:
This section was a bit tough for me to understand. I'm not sure the title represents the content either. What about something like this.
```
== Uptime, Heartbeat, and Kibana
For Uptime to work, it is important you use the same major versions of Heartbeat and Kibana.
For example, version 6.7 of Kibana will expect an index of `heartbeat-6*`,
while Kibana 7.0 requires an index of `heartbeat-7*` (containing documents from Heartbeat 7.0).

Reply:
@justinkambic - I see you implemented some changes here, but any thoughts on the title? I'm not sure "Begin using Uptime" accurately describes what this section is doing. This seems more about compatibility to me

Reply:
Yes I agree, I must've missed that when adding. Want me to push another commit?

Reply:
No worries. Yeah, that'd be great!

Reply:
See https://github.com/elastic/kibana/pull/31814/commits/7178ce3d8e4f9786e92992efa43ede5192205657.